cmd/lakectl/cmd/docs.go

```diff
@@ -60,6 +63,13 @@ func printOptions(buf *bytes.Buffer, cmd *cobra.Command) error {
 	}
 	parentFlags := cmd.InheritedFlags()
 	parentFlags.SetOutput(buf)
+
+	name := cmd.CommandPath()
+	if name == "lakectl" {
```

Comment:
Alternative solution, it can help more changes without looking for constant later.
```suggestion
	if cmd == rootCmd {
```

Reply:
Neat, thanks!